.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```

Comment:
This could be extended to run https://www.npmjs.com/package/lint-staged

Reply:
@GaryPWhite Exactly! No additional changes or set up are required on the developer side since the hooks get set up when you run yarn install.